app/services/backend/private-cloud/products.ts

```diff
@@ -147,6 +148,13 @@ export async function getPodUsageMetrics(licencePlate: string, environment: stri
   return response.data;
 }
 
+export async function getSubnetForEmerald(licencePlate: string, environment: string, cluster: string) {
```

Comment:
1. let's set default value for `enviroment` to `dev` here.
2. let's remove `cluster` from the arguments.